### DISCUSSION TITLE
Target Demographic

### DISCUSSION THREAD

**Discussion by Xansta** [Thorium-Sim/thorium-nova#16] [Ideas] [answered]
During the meeting, the target demographic was identified as Alex & family followed by others that are interested in starship bridge simulators. Coming from a background of Empty Epsilon, Artemis and (minimally) Starship Horizons, I'm biased towards the fully automated scenarios and missions with minimal or zero GM/flight director intervention. Having been a GM on both Artemis and Empty Epsilon including using their systems for user contribution of missions or scenarios, and having at least tried to make something work on Thorium classic, I can see that generally, Thorium relies heavily on the actions and guidance of the GM. EE and Artemis don't. I would like for Thorium Nova to allow us to have our cake and eat it too: namely give an excellent out of the box experience for people with little or no desire or experience with being a GM, but still able to experience the collaboration and sense of exploration of a starship bridge simulator *and* giving plenty of controls and customization features for those that want to use Thorium Nova in their space centers, in their live action role playing groups and/or just online with friends.

This may be quite a challenge to get both of these user groups and their sometimes contradictory use cases working, but I think it can be done.

**Comment by alexanderson1993** [comment, marked as answer]
Agreed. This is something that would set Thorium Nova apart from the other bridge simulators and give it a reason for existing in the first place.

**Reply by astrolamb-gaming** [reply]
I fully agree that having a GM should not be a requirement to play. This factors heavily into my own vision for the Flight DIrector UI as not only a GM tool, but also as a mission generation tool. I think it would be great for Thorium to support a repository of user-generated missions, which anyone could submit missions to, and to which all users would have access.

**Reply by alexanderson1993** [reply]
>  a repository of user-generated missions

This is part of the vision, and why https://thoriumsim.com now supports user accounts.
